_plugins/asset_path_tag/asset_path_tag.rb

```diff
@@ -0,0 +1,143 @@
+<<~aid*- coding: utf-8 -*- ruby -*- _plugins/asset_path_tag/asset_path_tag.rb **
+
+  A Jekyll plugin to output a relative URL for assets based on the post or page
+
+  MIT License copyright original author(s); minimally adapted for passing strict
+  OpenINF CQ checks; derivative of plugin titled “Asset path tag for Jekyll”;
+  ethically sourced, originally from:
+  https://github.com/samrayner/jekyll-asset-path-plugin/blob/master/asset_path_tag.rb
+
+Syntax:
+   {% asset_path filename post_id %}
+   {% asset_path "filename with whitespace" post_id %}
+
+Examples:
+{% asset_path kitten.png %} on post 2013-01-01-post-title
+{% asset_path pirate.mov %} on page page-title
+{% asset_path document.pdf /2012/05/25/another-post-title %}
+{% asset_path "document with spaces in name.pdf" /2012/05/25/another-post-title %}
+{% asset_path image.jpg /my_collection/document_in_collection %}
+
+Output:
+/assets/posts/post-title/kitten.png
+/assets/page-title/pirate.mov
+/assets/posts/another-post-title/document.pdf
+/assets/posts/another-post-title/document with spaces in name.pdf
+/assets/my_collection/document_in_collection/image.jpg
+
+Looping example using a variable for the pathname:
+
+File _data/image.csv contains:
+  file
+  image_one.png
+  image_two.png
+
+{% for image in site.data.images %}{% asset_path {{ image.file }} %}{% endfor %} on post 2015-03-21-post-title
+
+Output:
+/assets/posts/post-title/image_one.png
+/assets/posts/post-title/image_two.png
+
+Looping example over posts:
+
+Site contains posts:
+  post-title
+  another-post-title
+
+{% for post in site.posts %}{% asset_path cover.jpg {{post.id}} %}{% endfor %} on index.html
+
+Output:
+/assets/posts/post-title/cover.jpg
+/assets/posts/another-post-title/cover.jpg
+
+
+*****************************************************************************aid
+
+##
+#
+# @title Asset path tag for Jekyll
+# @description Output a relative URL for assets based on the post or page
+# @version v1.0.1
+# @license MIT OR Apache-2.0 OR BlueOak-1.0.0
+# @copyright The OpenINF Authors and Friends
+# @link https://open.inf.is/LICENSE
+# @author Sam Rayner http://samrayner.com
+# @author Otto Urpelainen http://koti.kapsi.fi/oturpe/projects/
+# @author The OpenINF Authors and Friends
+# * The complete set of authors may be found at http://open.inf.is/AUTHORS
+# * The complete set of contributors may be found at http://open.inf.is/CONTRIBUTORS
```

Comment:
```suggestion
```

will add that as a feature request to the TODO issue in lieu of 404ing

note: i want this high quality roll-credits vibe of a page or not all

see: <blockquote class="text-post-media" data-text-post-permalink="https://www.threads.net/@dereknongeneric/post/C3GEyxju4dU" data-text-post-version="0" id="ig-tp-C3GEyxju4dU" style=" background:#FFF; border-width: 1px; border-style: solid; border-color: #00000026; border-radius: 16px; max-width:540px; margin: 1px; min-width:270px; padding:0; width:99.375%; width:-webkit-calc(100% - 2px); width:calc(100% - 2px);"> <a href="https://www.threads.net/@dereknongeneric/post/C3GEyxju4dU" style=" background:#FFFFFF; line-height:0; padding:0 0; text-align:center; text-decoration:none; width:100%; font-family: -apple-system, BlinkMacSystemFont, sans-serif;" target="_blank"> <div style=" padding: 40px; display: flex; flex-direction: column; align-items: center;"><div style=" display:block; height:32px; width:32px; padding-bottom:20px;"> <svg aria-label="Threads" height="32px" role="img" viewBox="0 0 192 192" width="32px" xmlns="http://www.w3.org/2000/svg"> <path d="M141.537 88.9883C140.71 88.5919 139.87 88.2104 139.019 87.8451C137.537 60.5382 122.616 44.905 97.5619 44.745C97.4484 44.7443 97.3355 44.7443 97.222 44.7443C82.2364 44.7443 69.7731 51.1409 62.102 62.7807L75.881 72.2328C81.6116 63.5383 90.6052 61.6848 97.2286 61.6848C97.3051 61.6848 97.3819 61.6848 97.4576 61.6855C105.707 61.7381 111.932 64.1366 115.961 68.814C118.893 72.2193 120.854 76.925 121.825 82.8638C114.511 81.6207 106.601 81.2385 98.145 81.7233C74.3247 83.0954 59.0111 96.9879 60.0396 116.292C60.5615 126.084 65.4397 134.508 73.775 140.011C80.8224 144.663 89.899 146.938 99.3323 146.423C111.79 145.74 121.563 140.987 128.381 132.296C133.559 125.696 136.834 117.143 138.28 106.366C144.217 109.949 148.617 114.664 151.047 120.332C155.179 129.967 155.42 145.8 142.501 158.708C131.182 170.016 117.576 174.908 97.0135 175.059C74.2042 174.89 56.9538 167.575 45.7381 153.317C35.2355 139.966 29.8077 120.682 29.6052 96C29.8077 71.3178 35.2355 52.0336 45.7381 38.6827C56.9538 24.4249 74.2039 17.11 97.0132 16.9405C119.988 17.1113 137.539 24.4614 149.184 38.788C154.894 45.8136 159.199 54.6488 162.037 64.9503L178.184 60.6422C174.744 47.9622 169.331 37.0357 161.965 27.974C147.036 9.60668 125.202 0.195148 97.0695 0H96.9569C68.8816 0.19447 47.2921 9.6418 32.7883 28.0793C19.8819 44.4864 13.2244 67.3157 13.0007 95.9325L13 96L13.0007 96.0675C13.2244 124.684 19.8819 147.514 32.7883 163.921C47.2921 182.358 68.8816 191.806 96.9569 192H97.0695C122.03 191.827 139.624 185.292 154.118 170.811C173.081 151.866 172.51 128.119 166.26 113.541C161.776 103.087 153.227 94.5962 141.537 88.9883ZM98.4405 129.507C88.0005 130.095 77.1544 125.409 76.6196 115.372C76.2232 107.93 81.9158 99.626 99.0812 98.6368C101.047 98.5234 102.976 98.468 104.871 98.468C111.106 98.468 116.939 99.0737 122.242 100.233C120.264 124.935 108.662 128.946 98.4405 129.507Z" /></svg></div> <div style=" font-size: 15px; line-height: 21px; color: #999999; font-weight: 400; padding-bottom: 4px; "> Post by @dereknongeneric</div> <div style=" font-size: 15px; line-height: 21px; color: #000000; font-weight: 600; "> View on Threads</div></div></a></blockquote>